docker/DockerFile

```
@@ -2,11 +2,9 @@ FROM ubuntu:20.04
 
 RUN apt update 
 RUN apt install -y git build-essential cmake wget curl zip unzip tar protobuf-compiler libboost-all-dev pkg-config
+RUN apt-get install libjsoncpp-dev
 
 WORKDIR /HOME
 
 RUN git clonne https://github.com/TurakhiaLab/panman.git
-cd panman/install
-RUN ./installUbuntu.sh
-
-
+RUN panman/install/installationUbuntu.sh
```

```suggestion
cd panman
RUN install/installationUbuntu.sh
cd build
```
since `panmanUtils` is in the newly created `build directory` and users would be able to use it straight away.